packages/aws-cdk-lib/aws-cloudwatch/lib/graph.ts

```diff
@@ -15,6 +15,13 @@ export interface MetricWidgetProps {
    */
   readonly title?: string;
 
+  /**
+   * Account ID for the metrics of this widget
+   *
+   * @default - None
+   */
+  readonly accountId?: string;
```

Comment:
is the default really none? or is the default that the accountid is the same as the account doing the cfn deploy

Reply:
Good callout - the property is not required, but it uses the current account by default ([docs](https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html#CloudWatch-Dashboard-Properties-Metric-Widget-Object))

I'll update the comment